src/pyhf/pdf.py

```diff
@@ -152,7 +154,7 @@ def set_poi(self, name):
         self.poi_index = s.start
 
     def _register_paramset(self, param_name, paramset):
-        """allocates n nuisance parameters and stores paramset > modifier map"""
+        """Allocates n nuisance parameters and stores paramset > modifier map."""
```

Comment:
this doesn't read well to me. What is `n`? What does `paramset > modifier map` mean?